.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "workbench.colorCustomizations": {
+        "activityBar.background": "#550C45",
+        "titleBar.activeBackground": "#771161",
+        "titleBar.activeForeground": "#FFFBFE"
+    }
+}
```

Comment:
please don't add this since file, since it's not relevant for the site :)

Reply:
Oh  Sorry, I didn't know that😅 Just, I deleted it.  Can you merge now?